valhalla/baldr/graphconstants.h

```diff
@@ -201,6 +201,10 @@ constexpr uint32_t kMaxAddedTime = 255;
 // Elevation constants
 constexpr float kNoElevationData = 32768.0f;
 
+// Z-level range
+constexpr int8_t kMinEdgeZLevel = -6;
```

Comment:
based on https://taginfo.openstreetmap.org/keys/layer#values

Reply:
From https://wiki.openstreetmap.org/wiki/Key:layer:
> Values between -5 and 5 are generally used; 0 should not be used explicitly.

Should we use [-5;5] range instead?

Reply:
Could the doc-string contain that wiki link? I'm sure that would be helpful for anyone wondering what unit Z-level is.